.github/workflows/test.yml

```diff
@@ -141,6 +141,7 @@ jobs:
     runs-on: macos-latest
     env:
       _PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC: 0
+      HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: TRUE
```

Comment:
Wow, this variable name is a doozy. (I realize it's from the homebrew project - not something you came up with 🤷‍♂️).

Is the idea that since we're manually updating we can ignore the auto-update feature?

Reply:
Yes, as it transiently causes the build to fail as those dependent package installs are randomly failing for no reason (see last two CI failures on macos)